alcs-api/src/main.ts

```diff
@@ -55,14 +54,17 @@ const registerGlobalFilters = (app: NestFastifyApplication) => {
 };
 
 const registerCors = (app: NestFastifyApplication) => {
-  app.enableCors({
-    origin: [
-      config.get<string>('BASE_URL'),
-      config.get<string>('KEYCLOAK.AUTH_SERVER'),
-    ],
-  });
+  app.enableCors();
```

Comment:
my guess that this will break swagger authorization

Reply:
Fixed, add FRONTEND_ROOT to config and added these back